packages/components/badge/src/Badge.mdx

```diff
@@ -177,30 +174,14 @@ Tags are used to label or highlight items in the interface.
         />
       </Flex>
       <div>
-        <Tag tagType="positive">published</Tag>
+        <Badge variant="positive">published</Badge>
       </div>
     </Flex>
   </div>
 </>
 ```
 
-- **Muted** - this `tagType` is deprecated, please use `tagType` `secondary` instead
-
-### Tag sizes
```

Comment:
@suevalov sorry if I didn't notice in the PR - size property was removed from Badge?

Reply:
yep, it was. shouldn't it be?

Reply:
yeah, I think it should - it was just introduced a couple of weeks ago.

Reply:
ok, I see. I'm going to fix it then

Reply:
Thank you! 😊